app/controllers/documents/documents_help_controller.rb

```diff
@@ -7,18 +7,11 @@ class DocumentsHelpController < DocumentUploadQuestionController
     def show; end
 
     def send_reminder
+      doc_type = params[:doc_type].constantize.key
```

Comment:
I'm personally nervous when I see `.constantize` because unless we're careful, it can lead to [code execution.](http://gavinmiller.io/2016/the-safesty-way-to-constantize/)

I think this is a use that is vulnerable to "Command Injection" documented in that doc.

What do you think about using something other than `.constantize`, or am I misunderstanding the code flow here?

Reply:
Right now, we're using the passed doc type to get the key. Maybe instead of the doc_type, we can use the translation key and look it up directly from the translation file. Since this is an existing implementation and will require a refactor of this area of the code in more depth, I think we should file a  chore to come up with a better way to do this vs blocking this PR.